### PR TITLE
Remove obsolete "behat/common-contexts"

### PR DESCRIPTION
**Pull request by skeleton**
I broke the PIM ( :kissing_heart: ) when I upgraded phpunit, the obsolete package "behat/common-contexts" is not compatible with our new version. I remove it and add usefull methods in `features/Context/WebApiContext`.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

**Comment by ClementGautier** [comment]
GTM

File: features/Context/WebApiContext.php [hunk 1]
```diff
@@ -2,29 +2,54 @@
...
+
+    /** @var string */
     protected $url;
 
     /**
-     * {@inheritdoc}
+     * @param string  $baseUrl
+     * @param Browser $browser
      */
     public function __construct($baseUrl, Browser $browser = null)
```
Comment (aRn0D) [inline features/Context/WebApiContext.php]: Why $browser can be null ?

File: features/Context/WebApiContext.php [hunk 2]
```diff
@@ -82,6 +107,97 @@ public function theResponseShouldContainTheFollowingData(TableNode $table)
...
+    /**
+     * Checks that response body contains JSON from PyString.
+     *
+     * @param PyStringNode $jsonString
+     *
+     * @Then /^(?:the )?response should contain json:$/
+     */
+    public function theResponseShouldContainJson(PyStringNode $jsonString)
+    {
+        $etalon = json_decode($this->replacePlaceHolder($jsonString->getRaw()), true);
```
Comment (aRn0D) [inline features/Context/WebApiContext.php]: What **etalon** means plz?

File: features/Context/WebApiContext.php (hunk 1 above)
Comment (skeleton) [inline features/Context/WebApiContext.php]: I just copied the same constructor as the previous parent. Dunno why browser can be null

Comment (aRn0D) [inline features/Context/WebApiContext.php]: ok :)

Comment (skeleton) [inline features/Context/WebApiContext.php]: @aRn0D  in fact, Browser is always null https://github.com/akeneo/pim-community-dev/blob/master/features/Context/FeatureContext.php#L61
If you prefer, I can instance it directly

Comment (aRn0D) [inline features/Context/WebApiContext.php]: I think it could be better, no ?

File: features/Context/WebApiContext.php (hunk 2 above)
Comment (aRn0D) [inline features/Context/WebApiContext.php]: I guest it copy/paste from the old context ?

**Comment by aRn0D** [comment]
GTM once fixed

**Pull request merged**